src/hooks/useGetArticle.js

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react";
+import { getArticle } from "../api/fetch";
+
+export default function useGetArticle({ match }) {
+  const [article, setArticle] = useState(false);
+
+  const {
+    params: { articleId }
+  } = match;
+
+  function handleSubmit(event) {
+    const data = Object.fromEntries(new FormData(event.target).entries());
+
+    fetch(`/articles/${articleId}`, {
+      method: "PATCH",
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(data)
+    });
+  }
+
+  useEffect(() => {
+    getArticle(articleId).then(fetchedArticle => setArticle(fetchedArticle));
+    // eslint-disable-next-line
```

Comment:
This shouldn't be necessary if you update your eslint rules

Reply:
I haven't seen an easy way to turn it off. I also read, that it's inadvisable to do it. Didn't saved the source. :/